index.bs

```diff
@@ -2129,11 +2189,103 @@ an [=attribution trigger=] |trigger|:
     : [=aggregatable report/contributions=]
     :: The result of running [=create aggregatable contributions=] with |source| and |trigger|.
     : [=aggregatable report/serialized private state token=]
-    :: |trigger|'s [=attribution trigger/serialized private state token=]
+    :: null.
+    : [=aggregatable report/aggregation coordinator=]
+    :: |trigger|'s [=attribution trigger/aggregation coordinator=].
+    : [=aggregatable report/source registration time configuration=]
+    :: |trigger|'s [=attribution trigger/aggregatable source registration time configuration=].
+1. Return |report|.
+
+<h3 id="generating-randomized-null-reports">Generating randomized null reports</h3>
+
+To <dfn>obtain a randomized number of null reports</dfn> given a double |distributionParameter|:
+
+1. [=Assert=]: |distributionParameter| is between 0 and 1 ( both inclusive).
+1. Return a random non-negative integer from a geometric distribution with probability parameter
```

Comment:
Nit: we should probably specify how to sample from this (at the very least we should say the PMF is `p(1-p)^k`. Feel free to do this in a follow-up.
```suggestion
1. Return a random non-negative integer sampled from the geometric distribution with probability parameter
```

Reply:
Also maybe add an issue about possibly truncating this, although I think that might entail having an extra boolean to this algorithm that says whether the nulls will be accompanied by a real, so we can truncate the sum of real + fakes.